phpstan-persistence2.neon

```diff
@@ -16,6 +16,7 @@ parameters:
             path: lib/Doctrine/ORM/Tools/Console/ConsoleRunner.php
 
         - '/^Class Doctrine\\DBAL\\Platforms\\(PostgreSQL|SQLServer|SQLAnywhere)Platform not found\.$/'
+        - '/^Call to method \w+\(\) on an unknown class Doctrine\\DBAL\\Platforms\\(PostgreSQL|SQLServer|SQLAnywhere)Platform\.$/'
```

Comment:
Why is this happening? Doesn't PHPStan find the DBAL classes?

Reply:
Those classes don't exist in DBAL 3.

Reply:
Well, the SQLAnywhere one at least. The others have been added back in the meantim. I'll try to simplify the rule.

Reply:
Done.